chunk_test.go

```diff
@@ -636,3 +636,51 @@ func TestQueryArrowStreamDescribeOnly(t *testing.T) {
 		}
 	})
 }
+
+func TestRetainChunkWOHighPrecision(t *testing.T) {
+	runDBTest(t, func(dbt *DBTest) {
+		query := "select 0"
+
+		ctx := WithArrowBatches(context.Background())
+
+		var rows driver.Rows
+		var err error
+
+		// must use conn.Raw so we can get back driver rows (an interface)
+		// which can be cast to snowflakeRows which exposes GetArrowBatch
+		err = dbt.conn.Raw(func(x interface{}) error {
+			queryer, implementsQueryContext := x.(driver.QueryerContext)
+			assertTrueF(t, implementsQueryContext, "snowflake connection driver does not implement queryerContext")
```

Comment:
Same here, this is fine to do casting without this second parameter. Let it burn otherwise!